source/documentation/4.0/rspec-rails/file.Changelog.html

```diff
@@ -58,25 +58,65 @@
         <div class="clear"></div>
       </div>
 
-      <div id="content"><div id='filecontents'><h3>Development</h3>
+      <div id="content"><div id='filecontents'><h3>4.0.0.rc1 / 2020-03-13</h3>
```

Comment:
This is not what we want 🤔

Reply:
Nope :joy:, you might need to manually grab the commit after. The changelog in the release wasn't cleaned, my mistake :/

Reply:
Ahah. Yes it was the next one. It should be good now. 😊